projects/components/src/form/base-form-control.ts

```diff
@@ -73,6 +83,8 @@ export class BaseFormControl implements ControlValueAccessor, CanBeReadOnly {
 
     constructor(ngControl: NgControl) {
         this.id = idGenerator.generate();
+        this.descriptionId = idGenerator.generate();
```

Comment:
We don't really new a new ID, it could be
```ts
this.descriptionId = `${this.id}-description`;
this.errorsId =`${this.id}-errors`;
```

Also consider moving them to the initializer, makes for less typing and simpler constructors. Adding readonly will also prevent it from being reassigned

```ts

    /**
     * Auto generated ID for the input field.	     
    */
    readonly id = idGenerator.generate();
  /**
     * Auto generated ID for the description field.
     */
   readonly  descriptionId = `${this.id}-description;

    /**
     * Auto generated ID for the error field.
     */
    readonly errorsId = `${this.id}-errors`;
```

Reply:
Done, thanks!